src/mono/netcore/sample/wasm/browser/runtime.js

```diff
@@ -9,7 +9,8 @@ var Module = {
         config.fetch_file_cb = function (asset) {
             return fetch (asset, { credentials: 'same-origin' });
         }
-
+        MONO.mono_wasm_init_aot_profiler();
         MONO.mono_load_runtime_and_bcl_args (config);
     },
-};
+    runProfiler: true,
```

Comment:
Is this used anywhere? Ah, you're using it to call StopProfile.  Typically you'd call that from managed, in the case of the sample it could just be at the end of main.